### PR TITLE
Added timestamp & buttons to TFM presence

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
Added elapsed time to TFM presence
Added buttons for listening and Spotify of the track.

Evidence of testing: 
Evidence
Evidence

### CODE REVIEW

File: websites/T/TruckersFM/presence.ts [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
 
   presenceData.details = `${document.querySelector(".player-artist-text").textContent} - ${document.querySelector(".player-title-text").textContent}`;
   presenceData.state = document.querySelector(".live-name").textContent
     ? document.querySelector(".live-name").textContent
     : 'AutoDJ';
+	
+  presenceData.buttons = [
+	{
+		label: 'Tune into TFM',
+		url: 'https://truckers.fm'
```
Comment (ririxidev) [inline websites/T/TruckersFM/presence.ts:18]: image

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @NinjaLabs-Dev!
You should check out thing(s) that I requested to change in my review. Please look at them!

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

File: websites/T/TruckersFM/presence.ts (hunk 1 above)
Comment (ririxidev) [reply websites/T/TruckersFM/presence.ts:18]: You can redirect users to: ``https://truckers.fm/listen``

Comment (NinjaLabs-Dev) [reply websites/T/TruckersFM/presence.ts]: Changed to /listen

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by ririxidev — APPROVED** [review]
lgtm

**Pull request merged**